### PR TITLE
#5693: Correcting PHP notices for customers auth

### LINKED ISSUE #5693: Multiple PHP Notices when `CUSTOMERS_APPROVAL_AUTHORIZATION` is '2' and customer's not logged in
Seeing logs of the ilk:
```
[10-Apr-2023 14:21:31 Europe/Berlin] Request URI: /zc158/mice-c-1_9/microsoft-intellimouse-explorer-p-26.html, IP address: 127.0.0.1, Language id 1
#1  zen_check_show_prices() called at [C:\xampp\htdocs\zc158\includes\modules\meta_tags.php:267]
#2  require(C:\xampp\htdocs\zc158\includes\modules\meta_tags.php) called at [C:\xampp\htdocs\zc158\includes\templates\responsive_classic\common\html_header.php:25]
#3  require(C:\xampp\htdocs\zc158\includes\templates\responsive_classic\common\html_header.php) called at [C:\xampp\htdocs\zc158\index.php:42]
--> PHP Notice: Undefined index: customers_authorization in C:\xampp\htdocs\zc158\includes\functions\functions_lookups.php on line 203.

[10-Apr-2023 14:21:31 Europe/Berlin] Request URI: /zc158/mice-c-1_9/microsoft-intellimouse-explorer-p-26.html, IP address: 127.0.0.1, Language id 1
#1  require(C:\xampp\htdocs\zc158\includes\modules\meta_tags.php) called at [C:\xampp\htdocs\zc158\includes\templates\responsive_classic\common\html_header.php:25]
#2  require(C:\xampp\htdocs\zc158\includes\templates\responsive_classic\common\html_header.php) called at [C:\xampp\htdocs\zc158\index.php:42]
--> PHP Notice: Undefined variable: meta_products_price in C:\xampp\htdocs\zc158\includes\modules\meta_tags.php on line 290.

[10-Apr-2023 14:21:31 Europe/Berlin] Request URI: /zc158/mice-c-1_9/microsoft-intellimouse-explorer-p-26.html, IP address: 127.0.0.1, Language id 1
#1  require(C:\xampp\htdocs\zc158\includes\modules\attributes.php) called at [C:\xampp\htdocs\zc158\includes\modules\pages\product_info\main_template_vars.php:50]
#2  require(C:\xampp\htdocs\zc158\includes\modules\pages\product_info\main_template_vars.php) called at [C:\xampp\htdocs\zc158\includes\templates\responsive_classic\common\tpl_main_page.php:178]
#3  require(C:\xampp\htdocs\zc158\includes\templates\responsive_classic\common\tpl_main_page.php) called at [C:\xampp\htdocs\zc158\index.php:94]
--> PHP Notice: Undefined index: customers_authorization in C:\xampp\htdocs\zc158\includes\modules\attributes.php on line 154.

[10-Apr-2023 14:21:31 Europe/Berlin] Request URI: /zc158/mice-c-1_9/microsoft-intellimouse-explorer-p-26.html, IP address: 127.0.0.1, Language id 1
#1  require(C:\xampp\htdocs\zc158\includes\modules\attributes.php) called at [C:\xampp\htdocs\zc158\includes\modules\pages\product_info\main_template_vars.php:50]
#2  require(C:\xampp\htdocs\zc158\includes\modules\pages\product_info\main_template_vars.php) called at [C:\xampp\htdocs\zc158\includes\templates\responsive_classic\common\tpl_main_page.php:178]
#3  require(C:\xampp\htdocs\zc158\includes\templates\responsive_classic\common\tpl_main_page.php) called at [C:\xampp\htdocs\zc158\index.php:94]
--> PHP Notice: Undefined index: customers_authorization in C:\xampp\htdocs\zc158\includes\modules\attributes.php on line 154.
```

Noting that the log from `meta_tags.php` is since it's receiving `false` from the function `zen_check_show_prices` and the noted variable isn't pre-set on that path.

### PR DESCRIPTION
**Pull request by lat9**
Fixes #5693.

### CODE REVIEW

**Review by brittainmark — COMMENTED** [review]
That was some logic feat! looks good to me

File: includes/functions/functions_lookups.php [hunk 1]
```diff
@@ -192,25 +192,33 @@ function zen_run_normal(): bool
...
 
 /**
- *  Look up whether to show prices, based on customer-authorization levels
+ * Look up whether to show prices, based on customer-authorization levels
+ *
+ * Prices are NOT shown if
+ *
+ * 1. The site's 'Store Status' is '1' (Showcase no prices).
+ * 2. 'Customer Shop Status - View Shop and Prices' is '2' (must login to see prices) and a customer is not logged in.
+ * 3. 'Customer Approval Status - Authorization Pending' is '1' (Must be Authorized to Browse) or '2' (May browse but no prices unless Authorized) and eitherper
```
[the remark this answers is not in the document]
Comment (lat9) [reply includes/functions/functions_lookups.php]: Thanks for the review!  Unraveling that logic had my head spinning!

**Review by brittainmark — APPROVED** [review]
looks good to me

**Comment by scottcwilson** [comment]
Nice simplification.

**Pull request merged**